nyc_taxis/challenges/default.json

```diff
@@ -936,14 +936,37 @@
           "operation": "avg_passenger_count_esql_segment_partitioning",
           "clients": 1,
           "warmup-iterations": 10,
-          "iterations": 50
+          "iterations": 50,
+          "tags": ["inlinestats"]
```

Comment:
Does adding a tag on an existing challenge have unintended consequences (like the historical for the data resetting or something)?